### DISCUSSION TITLE
Does the update function not support CronWithSeconds?

### DISCUSSION THREAD

**Discussion by Chen840302** [go-co-op/gocron#188] [Q&A] [answered]
This is my code,
I need to change my job to cronExpression

```javascript
	job ,_ := s.Every(3).Seconds().Do(job1.task, "123")
	s.StartAsync()

	time.Sleep(time.Second * 6)
	s.Job(job).CronWithSeconds("*/3 * * * * *").Update()
```

But it will show the following error message

![image](https://user-images.githubusercontent.com/68727360/121001476-e898b380-c7bd-11eb-9f46-ced4df41e339.png)

**Comment by JohnRoesler** [comment, marked as answer]
It’s likely that I missed adding update support for the cron functionality. I’ll check that out when I have time again (hopefully soon)

**Reply by JohnRoesler** [reply]
This is resolved! https://github.com/go-co-op/gocron/issues/193
